app/src/amazonWebview/java/org/mozilla/focus/webview/FirefoxAmazonWebView.kt

```diff
@@ -50,12 +50,6 @@ internal class FirefoxAmazonWebView(
         setOnLongClickListener(it)
     }
 
-    override fun onCreateInputConnection(outAttrs: EditorInfo): InputConnection {
-        val connection = super.onCreateInputConnection(outAttrs)
-        outAttrs.imeOptions = outAttrs.imeOptions or ViewUtils.IME_FLAG_NO_PERSONALIZED_LEARNING
```

Comment:
It's a flag to suggest to the keyboard not to learn from input (e.g. autocomplete suggestions) in case you're typing in private content like a password or engagement rings or...